src/pages/admin/reports/components/funds-expenditures-section/components/common/translate-disclaimer-modal/TranslateDisclaimerModal.module.scss

```
@@ -0,0 +1,72 @@
+    :global(.modal-header) {
+        margin-bottom: 0;
+        padding: 0;
+    }
+
+    :global(.modal-header .modal-title-wrapper) {
+        text-align: left;
+    }
+
+    :global(.modal-header-text) {
+        @include type.h4-regular;
+        color: c.$blue-900;
```

Stylelint (17.11.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)